### PR TITLE
Added how to move sidecar button

### PR DESCRIPTION
**Pull request by GiacomoLaw**
Fixes https://github.com/gitterHQ/sidecar/issues/53

Added section on how to move sidecar "Open Chat" button

### CODE REVIEW

**Comment by mydigitalself** [comment]
Doesn't this just move the button?

**Comment by GiacomoLaw** [comment]
Yes. It does that because people may have a tawk.to widget or something like that in the bottom right.

**Comment by mydigitalself** [comment]
@GiacomoLaw I guess my point is that this doesn't move **SIDECAR** it only moves the button, so maybe it might be better to describe it like that in the PR and - more importantly - in the README otherwise people may this it moves the position of the chat window, which is really what we talk about when we mean sidecar.

**Comment by GiacomoLaw** [comment]
Fixed it. Is that okay?

**Comment by mydigitalself** [comment]
Yip. @MadLittleMods is this the correct approach btw?

**Comment by GiacomoLaw** [comment]
Why are the checks failing? :confused:

File: README.md [hunk 1]
```diff
@@ -30,6 +30,14 @@ var myChat = new Sidecar({
 });
 ```
 
+## Change position of sidecar button.
+```css
+.gitter-open-chat-button {
+  right: 250px;
+}
+```
+This will move the sidecar button to the right.
```
Comment (MadLittleMods) [inline README.md]: Should be 

>This will move the sidecar button 250px from the right.

or

>This will move the sidecar button 250px to the left

**Comment by GiacomoLaw** [comment]
Updated it. Do you want me to add how to put it on the left too?

File: README.md (hunk 1 above)
Comment (GiacomoLaw) [inline README.md]: Okay, sure. Done! 😄

**Comment by MadLittleMods** [comment]
@GiacomoLaw No, there are so many possible things to document but it is just CSS. You can style Sidecar in any way. Another common use case to document is changing the z-index so that the chat appears in front of a fixed header but we can cover that in a separate PR.

**Comment by GiacomoLaw** [comment]
Should I delete the branch?

**Pull request closed without merging**